azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -591,6 +593,24 @@ private void ajaxScheduleCronFlow(final HttpServletRequest req,
       return;
     }
 
+    final boolean hasFlowTrigger;
+    try {
+      hasFlowTrigger = this.projectManager.hasFlowTrigger(project, flow);
+    } catch (final Exception ex) {
+      logger.error(ex);
+      ret.put("status", "error");
+      ret.put("message", String.format("Flow %s.%s cannot be scheduled: %s", projectName,
```

Comment:
Shouldn't the error message be `Error while looking for flow Trigger`?

Reply:
updated